axelor-sale/src/main/resources/views/SaleConfig.xml

```diff
@@ -17,7 +17,7 @@
       <field name="saleOrderInAtiSelect"/>
       <field name="defaultValidityDuration" domain="self.applicationTypeSelect = 1"
         form-view="duration-form" grid-view="duration-grid" colSpan="4"/>
-      <field name="defaultProjectHoldBackPercentage"/>
```

Comment:
Please do this change in another ticket, it is a anomaly that should be fixed in 8.1